### PR TITLE
[ZN-366] replace miragejs with msw

### PR DESCRIPTION
**Pull request by sylwiawajgel**

### CODE REVIEW

**Review by barto-tsh — APPROVED** [review]

**Review by knoszczynski — APPROVED** [review]

File: public/mockServiceWorker.js [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+
+async function handleRequest(event, requestId) {
+  const client = await resolveMainClient(event)
+  const response = await getResponse(event, client, requestId)
+
+  // Send back the response clone for the "response:*" life-cycle events.
+  // Ensure MSW is active and ready to handle the message, otherwise
+  // this message will pend indefinitely.
+  if (client && activeClientIds.has(client.id)) {
+    ;(async function () {
```
Comment (qh4r) [inline public/mockServiceWorker.js:148]: colon at the start?

File: public/mockServiceWorker.js [hunk 2]
```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
```
Comment (qh4r) [inline public/mockServiceWorker.js:1]: I assume all of this is a paste

**Review by qh4r — APPROVED** [review]

**Comment by knoszczynski** [comment]
@sylwiawajgel Cypress tests are failing, please fix it. In addition you have some conflicts?

File: public/mockServiceWorker.js (hunk 2 above)
Comment (sylwiawajgel) [reply public/mockServiceWorker.js:1]: This file is generated by CLI (https://mswjs.io/docs/getting-started/integrate/browser#setup)

**Pull request merged**